frontend/src/components/auth/ResetPasswordForm.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+    <VContent>
+        <VFlex
+                lg6
+                md6
+                sm12
+                xs12
+        >
+            <VContainer>
+                <VCardText>
+                    <VSubheader class="reset-password-form-header">
+                        Welcome to Metrica!
+                    </VSubheader>
+                    <VForm ref="form">
+                        <VSubheader class="reset-password-form-label">
+                            Email
+                        </VSubheader>
+                        <VTextField
+                                name="email"
+                                class="reset-password-form-input"
+                                v-model="email"
+                                solo
+                                type="text"
+                                :rules="emailRules"
+                                required
+                        />
+                    </VForm>
+                </VCardText>
+                <VCardActions>
+                    <VBtn @click="onResetPassword" class="reset-password-form-button mt-3" color="#3C57DE">
+                        Send Password Reset Link
+                    </VBtn>
+                </VCardActions>
+                <v-alert class="status-success" v-if="status==='success'" type="success">
```

Comment:
probably better avoid using v-if-else here. You can define computed prop like `showAlert` that will be true, when status not empty. Class value also can be move to computed prop and define it's value based on status, same for type